agreement/encoding_test.go

```diff
@@ -34,3 +34,13 @@ func TestEmptyEncoding(t *testing.T) {
 	var b bundle
 	require.Equal(t, 1, len(protocol.Encode(&b)))
 }
+
```

Comment:
I think you can delete this, because msgp now auto-generates calls to `RunEncodingTest` of every generated type.